website/junos.erb

```diff
@@ -12,6 +12,9 @@
           <li<%= sidebar_current("docs-junos-data-source-interface") %>>
             <a href="/docs/providers/junos/d/interface.html">junos_interface</a>
           </li>
+          <li<%= sidebar_current("docs-junos-data-source-system-information") %>>
+            <a href="/docs/providers/junos/d/system_information.html">system_information</a>
```

Comment:
Name of link need to be `junos_system_information`

Reply:
I updated as you specified. Thanks for the catch!